packages/react/src/ActionList/ActionList.module.css

```diff
@@ -186,7 +186,8 @@
     }
 
     /* active state [aria-current] */
-    &:where([data-active]) {
+    &:where([data-active]),
+    &:where([aria-selected='true']) {
```

Comment:
The implications of this may stem further than just the small change I want, but is a result of mixing React & non-React together (e.g. web-components)

In my case, the `aria-selected` role is added by the `tab-container` element so it never makes it back into the React Component. I could also add a handler to my own code to track selection changes and update the props accordingly, but I'd rather avoid that if I don't have to (a CSS only solution is much more performant). if we decide the risk of adding this is too high or it breaks to many things, I can undo this part of the change.